inference/inference-core/src/commonMain/kotlin/io/kinference.core/operators/math/BiasGelu.kt

```diff
@@ -109,11 +91,89 @@ class BiasGeluVer1(name: String, attributes: Map<String, Attribute<Any>> = empty
                 }
             }
         }
+
+        suspend fun computeGelu(input: FloatNDArray, bias: FloatNDArray): MutableFloatNDArray {
+            val output = MutableFloatNDArray(input.strides)
+
+            val blocks = input.array.blocks
+            val biasBlocks = bias.array.blocks
+            val outputBlocks = output.array.blocks
+
+            val blockSize = input.array.blockSize
+            val batchSize = run {
```

Comment:
Actually, all of these block size functions follow the same pattern. Maybe you can put this as a function somewhere in utils instead(and docs, please)?